doc/api/matplotlib_configuration_api.rst

```diff
@@ -12,26 +12,24 @@ The top level :mod:`matplotlib` module
 
     An instance of :class:`RcParams` for handling default matplotlib values.
 
-.. autofunction:: rc
-
-.. autofunction::rcdefaults
-
-.. autofunction::rc_file
+.. autodata:: rc_context
```

Comment:
See http://www.sphinx-doc.org/en/stable/ext/autodoc.html?highlight=autodata#directive-autofunction, but I guess this technically still qualifies as a function.